packages/cli-clean/src/clean.ts

```diff
@@ -1,11 +1,10 @@
-import {getLoader} from '@react-native-community/cli-tools';
+import {getLoader, prompt} from '@react-native-community/cli-tools';
 import type {Config as CLIConfig} from '@react-native-community/cli-types';
 import chalk from 'chalk';
 import execa from 'execa';
 import {existsSync as fileExists, rmdir} from 'fs';
 import os from 'os';
 import path from 'path';
-import prompts from 'prompts';
```

Comment:
Can we get rid of prompts dependency from `cli-clean`? And other packages, if feasable

Reply:
Done